src/sprinkles.css.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/extensions */
+import {defineProperties, createSprinkles} from '@vanilla-extract/sprinkles';
+import {vars} from './skins/skin-contract.css';
+
+type ColorKeys = keyof typeof vars.colors;
+type ColorValues = typeof vars.colors[ColorKeys];
+
+const colors: Array<ColorValues | 'transparent'> = [...Object.values(vars.colors), 'transparent'];
+
+const colorProperties = defineProperties({
+    properties: {
+        color: colors,
+        background: colors,
+        backgroundColor: colors,
+    },
+});
+
+const sizes = [0, 2, 4, 8, 12, 16, 24, 32, 40, 48, 56, 64, 72, 80] as const;
+
+const commonProperties = defineProperties({
```

Comment:
from the documentation

```
 conditions: {
    mobile: {},
    tablet: { '@media': 'screen and (min-width: 768px)' },
    desktop: { '@media': 'screen and (min-width: 1024px)' }
  }
```
  
  i think this is also interesting

Reply:
I still need to figure out how to work with mediaqueries, because until now mistica media queries have been configurable with context, but I don't know how to make them configurable and at the same time be able to use them in vanilla extract / sprinkles. (css doesn't allow variables `@media` :disappointed: )

Reply:
I've added a TODO to the PR list